static/app/views/starfish/components/samplesTable/spanSamplesTable.tsx

```diff
@@ -26,36 +27,54 @@ const COLUMN_ORDER: TableColumnHeader[] = [
   },
   {
     key: 'p95_comparison',
-    name: 'Compared to P95',
+    name: 'Compared to baseline',
     width: 200,
   },
 ];
 
 type SpanTableRow = {
   op: string;
-  'span.self_time': number;
-  span_id: string;
-  timestamp: string;
   transaction: {
     id: string;
     'project.name': string;
     timestamp: string;
     'transaction.duration': number;
   };
-  'transaction.id': string;
-};
+} & SpanSample;
 
 type Props = {
   data: SpanTableRow[];
   isLoading: boolean;
   p95: number;
+  highlightSpanId?: string;
```

Comment:
I think this should be called `highlightedSpanId` or something similar, with an `ed` suffix. `highlightSpanId` implies it's some kind of function, or handler, or something!

Reply:
That's a good point, will make the change!